app/assets/javascripts/service_banner.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener("turbolinks:load", function() {
+    let buttons = document.getElementsByClassName('close-service-notification');
```

Comment:
The sprockets pipeline seems to have a problem with ES6. Need to either convert or tweak the pipeline to make it work. IN POM-511 I just converted to ES5.

Reply:
Done